server/database/postgresql/queries.js

```diff
@@ -0,0 +1,8 @@
+const db = require('./client.js');
+
+const getRecordById = (id) => {
+  const q = 'SELECT * FROM hostels WHERE id = $1';
```

Comment:
You use template literals in some of your other requests. Could you do the same here? 😃

Reply:
The Airbnb style linter throws junk at me if I try. :D